www/source/partials/docs/examples/kubernetes_hab_bastion/hab-bastion-consumer.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: cockroachdb
+spec:
+  selector:
+    matchLabels:
+      app: cockroachdb
+  serviceName: cockroachdb
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: cockroachdb
+    spec:
+      terminationGracePeriodSeconds: 10
+      securityContext:
+        fsGroup: 42
+      containers:
+      - name: cockroachdb
+        image: irvingpop/cockroach:latest
+        args:
+        - --peer
+        - hab-bastion
```

Comment:
Does this end up working in practice when the bastion pod gets rescheduled? I don't think that DNS information persists within the system; it gets resolved to a concrete IP address early on, and that gets used thereafter.

Reply:
the way I do it the DNS name maps to a service which acts as an LB, so it doesn't matter if the bastion pod itself gets rescheduled.  in admittedly limited testing that seemed sane enough

Reply:
:thinking: I'm not sure what the underlying details of Kubernetes Service IP addressing are, but this might be worth keeping an eye on in the future.